integreat_cms/cms/constants/roles.py

```diff
@@ -75,15 +75,16 @@
     "change_imprintpage",
     "change_pushnotification",
     "change_user",
-    "change_chatmessage",
+    "change_organization" "change_chatmessage",
```

Comment:
Is is sufficient to add the permissions in [here](https://github.com/digitalfabrik/integreat-cms/blob/184f0302cd573e4c2ff19f9b343ea9013e84fc71/integreat_cms/cms/migrations/0002_roles.py) and not to create a new migration file?

Reply:
Update: Just adding it in roles.py does the trick already. I tested it without and with the changes by logging in as as management user. Before the changes you can only view organizations, afterwards you can also delete and create them. I feel like it does it since it is used in the update roles migration :question:

Reply:
Yeah, the problem is that this only works for newly set up systems. Django keeps track of which migrations were already applied and on production systems which already have the migration `0002_roles` applied, nothing will change. You can simulate this behavior by:
1. Check out the `develop` branch
2. Prune the database
3. Run the server
4. Stop it again
5. Check out the `feature/better_organization_management` branch
6. Run the server again

Then, it should have the `0002_roles` migration applied, but in its develop-state, so without the additional permission...

Reply:
Ah that makes sense, I will put it in a new migration file :+1:

Reply:
Alright, it should be fixed now